trello_client.py

```diff
@@ -0,0 +1,59 @@
+import os
+import requests
+
+base_url = "https://api.trello.com/1"
+board_id = os.getenv('BOARD_ID')
+key = os.getenv('API_KEY')
+token = os.getenv('API_TOKEN')
+todo_list_id = os.getenv('TODO_LIST_ID')
+doing_list_id = os.getenv('DOING_LIST_ID')
+done_list_id = os.getenv('DONE_LIST_ID')
```

Comment:
Just a heads up for module 3: you will want to load these *after* the test has set up the environment variables but currently you only read the env vars once when the file is first imported. 
So to get the tests to work you will need to change these into function instead of constants. Or you could set up the variables as part of app.py's `create_app` function, which exercise 3 asks you to make.